File: Coherent-Moving-Cluster/PythonSource/Test_Driver.py
from TrajectoryParser import TrajectoryParser
from CMC import CMC

if __name__ == '__main__':
    parser = TrajectoryParser("resource/testing/testing.csv")
    traj_set = parser.get_traj_set()
    res = CMC.cm_clustering(traj_set, 3, 3, 10)
    for conv in res:
        print(conv.toString())

File: Coherent-Moving-Cluster/JavaSource/coherent-moving-cluster/coherent-moving-cluster/src/main/Test_Driver.java
package main;

import java.util.List;

public class Test_Driver {


    public static void main(String[] args) throws Exception {

        TrajectoryParser parser = new TrajectoryParser("resource/data.csv");
        List<Trajectory> traj_set = parser.get_traj_set();
        List<Convoy> res = CMC.cm_clustering(traj_set, 4, 4, 5);

        for (Convoy convoy : res)
            System.out.println(convoy);

    }

}
